frontend/src/components/CreateCredentialModal.test.tsx

```diff
@@ -0,0 +1,76 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { CreateCredentialModal } from './CreateCredentialModal'
+
+// mock for the translation function
+jest.mock('../lib/acm-i18next', () => ({
+  useTranslation: () => ({
+    t: (key: string) => {
+      if (key === 'Add credential') return 'Add credential'
+      return key
+    },
+  }),
+}))
```

Comment:
Tests pass without this. We already have translation set up for tests:
https://github.com/stolostron/console/blob/main/frontend/src/setupTests.ts#L263